composer.json

```diff
@@ -19,22 +19,21 @@
     ],
     "require" : {
         "php": "^7.4 || ^8.0",
-        "mouf/magic-query": "^1.4.3",
+        "mouf/magic-query": "2.0.x-dev",
```

Comment:
Once everything works:
1. Merge changes to `mouf/magic-query`  if there are any
2. Release a new  `mouf/magic-query` version (2.0.0)
3. Require this new version here